src/Development/IDE/Core/Rules.hs

```diff
@@ -102,15 +103,17 @@ getDependencies file = fmap transitiveModuleDeps <$> use GetDependencies file
 getAtPoint :: NormalizedFilePath -> Position -> Action (Maybe (Maybe Range, [T.Text]))
 getAtPoint file pos = fmap join $ runMaybeT $ do
   opts <- lift getIdeOptions
-  spans <- useE GetSpanInfo file
-  return $ AtPoint.atPoint opts spans pos
+  (spans, mapping) <- useE GetSpanInfo file
+  !pos' <- MaybeT (return $ fromCurrentPosition mapping pos)
```

Comment:
fwiw this is `hoistMaybe` in `errors`. Haven’t checked if we already depend on that (transitively). I’ll leave it up to you whether you want to use it (potentially adding the dependency) or not.

Reply:
I'll leave it like this to be consistent with upstream ghcide.